CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### [Next]
 
+- support for unicode chars while parsing files.
```

Comment:
Limited support. 
https://doc.rust-lang.org/std/primitive.str.html#method.chars 
We're respecting UTF-8 scalar values, but aren't taking Unicode into consideration fully.